go/chat/pager/pager.go

```diff
@@ -0,0 +1,145 @@
+package pager
```

Comment:
Is the plan to have the pager used in other places, or is it only for local storage's internal paging?